community/releases/release-manager-runbook/index.md

```diff
@@ -269,13 +269,12 @@ channel to let them know that a new patch is available.
     1. Each Spin CLI release is tied to a version of Gate. To ensure
     compatibility, regenerate the Gate Client API.
 
-    1. From the Gate repository, check out the release branch and follow the
-    [instructions](https://github.com/spinnaker/spin/blob/master/CONTRIBUTING.md#updating-the-gate-api)
-    for updating the generated Gate Client API. Cherry-pick the Gate Client API
-    changes onto the Spin CLI release branch. As of writing, the Swagger
-    Codegen CLI uses 2.3.1; you can get that JAR [here](https://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/2.3.1/swagger-codegen-cli-2.3.1.jar).
-    If using a different version, you can try modifying the version parameters
-    in the URL.
+    1. From the Gate repository, check out the release branch and generate the `swagger/swagger.json` file (it's not under source control):
+    ```
+    ./swagger/generate_swagger.sh
+    ```
+    
+    1. From the spin-cli repository, check out the release branch (release branches from gate and spin-cli must match) and follow the [instructions](https://github.com/spinnaker/spin/blob/master/CONTRIBUTING.md#updating-the-gate-api) in that repo to update the gate client. This involves creating and merging a PR to spin-cli release branch with the updated gate client.
```

Comment:
Can we be consistent with the casing of Spin CLI and Gate?

Reply:
It looks like these are all references to the respective repositories, so we should use the repo names: "spin" and "gate"